src/coupler/surface_flux.F90

```diff
@@ -494,6 +502,37 @@ subroutine surface_flux_1d (                                           &
        rough_mom, rough_heat, rough_moist, w_atm,          &
        cd_m, cd_t, cd_q, u_star, b_star, avail             )
 
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+!!!!!!! added by mp586 for 10m winds and 2m temperature add mo_profile()!!!!!!!!
+
+
+  zrefm = 10. !want winds at 10m
+  zrefh = 2.  !want temp at 2m
+
+  call mo_profile( zrefm, zrefh, z_atm,   rough_mom, &
+       rough_heat, rough_moist,          &
+       u_star, b_star, q_star,        &
+       ex_del_m, ex_del_h, ex_del_q, avail  )
+
+
+! adapted from https://github.com/mom-ocean/MOM5/blob/99168b44ab45f4f5b4fa2544a0c3f644f0afb666/src/coupler/surface_flux.F90, L 1984 ff 
```

Comment:
I don't think this is the right link, as I can't find similar code there. I think you mean this one:
https://github.com/mom-ocean/MOM5/blob/3702ad86f9653f4e315b98613eb824a47d89cf00/src/coupler/flux_exchange.F90#L1932

Reply:
If you could update this comment then that would be awesome.